keps/sig-node/2400-node-swap/README.md

```diff
@@ -117,6 +118,20 @@ support to nodes in a controlled, predictable manner so that Kubernetes users
 can perform testing and provide data to continue building cluster capabilities
 on top of swap.
 
+This KEP aims to
+introduce basic swap enablement and leave further extensions to follow-up KEPs.
+This way Kubernetes users / vendors would be able to use swap in a basic manner
+quickly while extensions would be brought to discussion in dedicated KEPs that
+would progress in the meantime.
+
+For example, to achieve this goal, this KEP does not introduce any APIs
+that allow customizing how the feature behaves, but instead only determines 
+whether the feature is enabled or disabled.
+Instead, the behaviour is automatic and implicit that requires minimum user
```

Comment:
As we are trying to GA this, do we have any feedback on the calculations we offered here? So far the only feedback I was getting was on Unlimited. 


Also:
1. @tallclair once this is GA, InPlace pod update KEP will need to agjust these calculations as well. Are you tracking this?
2. The calculation becomes de-facto the standard. So it will be hard to change. Do we have some kind of conformance testing we can do?

Reply:
>As we are trying to GA this, do we have any feedback on the calculations we offered here? So far the only feedback I was getting was on Unlimited.

I feel the same - the only feedback I was getting is that some people want to bring back UnlimitedSwap, which was removed. On the other hand, UnlimitedSwap is indeed more dangerous, as a single pod can bring instability to the entire node.

Personally I think that it's reasonable to bring UnlimitedSwap back as long as we properly document that it's dangerous to use, but I don't have a strong opinion on this.

>@tallclair once this is GA, InPlace pod update KEP will need to agjust these calculations as well. Are you tracking this?

Yeah that's a good point. Thanks for pointing that out.
FWIW @tallclair I'll gladly help with implementing this.

> The calculation becomes de-facto the standard. So it will be hard to change. Do we have some kind of conformance testing we can do?

Yes, we do.
I've presented "swap-conformance" tests that thoroughly cover this.

[1] https://testgrid.k8s.io/sig-node-kubelet#kubelet-swap-conformance-fedora-serial
[2] https://testgrid.k8s.io/sig-node-kubelet#kubelet-swap-conformance-ubuntu-serial